tsl/src/fdw/scan_exec.c

```diff
@@ -476,6 +479,7 @@ fdw_scan_explain(ScanState *ss, List *fdw_private, ExplainState *es, TsFdwScanSt
 
 		if (ts_guc_enable_remote_explain)
 		{
+			Assert(fsstate);
```

Comment:
since fsstate can be NULL according to comment on line 453 this should be part of the if check otherwise the next line will segfault, asserts that should actually be proper check is a very common source of bugs

Reply:
If `ts_guc_enable_remote_explain` is enabled then even for EXPLAIN cases, we should be setting up `fsstate`. So asserting that it's present is needed, no? @svenklemm